manifests/params.pp

```diff
@@ -116,6 +116,6 @@
   $puppi_helper = 'standard'
   $debug = false
   $audit_only = false
-  $noops = undef
+  $noops = false
```

Comment:
If we set this to false the relevant resources will have always noop=false also when puppet in run with --noop option.
That's something that we don't want. (going to fix your other PR I just merged)